spec/forms/state_file/id_disability_form_spec.rb

```diff
@@ -325,4 +325,72 @@
       end
     end
   end
+
+  describe "#initialize" do
+    let(:primary_disabled) { "unfilled" }
+    let(:spouse_disabled) { "unfilled" }
+    let(:form) { described_class.new(intake, {})}
+
+    before do
+      intake.update(primary_disabled: primary_disabled, spouse_disabled: spouse_disabled)
+    end
+
+    context "mfj_disabled" do
+      before do
+        allow(intake).to receive(:show_mfj_disability_options?).and_return true
+      end
+
+      context "unfilled" do
+        it "returns a hash with the mfj_disabled attribute not populated" do
+          expect(form.mfj_disability).to eq nil
+        end
+      end
+
+      context "primary_disabled" do
+        let(:primary_disabled) { "yes" }
+        let(:spouse_disabled) { "no" }
+
+        it "returns a hash with the mfj_disabled attribute populated 'primary'" do
+          expect(form.mfj_disability).to eq "primary"
+        end
+      end
+
+      context "spouse_disabled" do
+        let(:primary_disabled) { "no" }
+        let(:spouse_disabled) { "yes" }
+
+        it "returns a hash with the mfj_disabled attribute populated 'spouse'" do
+          expect(form.mfj_disability).to eq "spouse"
+        end
+      end
+
+      context "both disabled" do
+        let(:primary_disabled) { "yes" }
+        let(:spouse_disabled) { "yes" }
+
+        it "returns a hash with the mfj_disabled attribute populated 'both'" do
+          expect(form.mfj_disability).to eq "both"
+        end
+
+        context "if changing to new disability state (form gets initialized from update)" do
+          before do
+            form.mfj_disability = "primary"
+          end
+
+          it "should preserve the updated disability state, instead of overwriting from db" do
+            expect(form.mfj_disability).to eq "primary"
+          end
+        end
```

Comment:
added the relevant spec here to [address the concern about the ability to update to a new mfj_disability attribute](https://github.com/codeforamerica/vita-min/pull/5933/files#r2047838597), but lmk if there's a better place to put it. I stuck it in the `both` section just to show a change, can be part of any spec (except the unfilled one).